### DISCUSSION TITLE
Formik integration

### DISCUSSION THREAD

**Discussion by jannikbuschke** [mantinedev/mantine#1732] [Show]
Not sure if this is the right channel but if anyone is interested in a formik integration here you go https://github.com/jannikbuschke/formik-mantine
I use Formik a lot and even though Mantine provides a form state solution, I have several older apps that need to maintained/migrated and I am not yet sure if mantines solution is ready for me yet , or if I am ready to migrate. If anyone else is in the same shoes pls checkout the integration library above. Hope this is not off topic here

https://codesandbox.io/s/formik-mantine-showroom-46h1jx?file=/src/App.tsx
![image](https://user-images.githubusercontent.com/5894881/177637339-06000fc6-f0b6-4361-8259-c052c12b01da.png)

**Comment by sillypnk** [comment]
> Not sure if this is the right channel but if anyone is interested in a formik integration here you go https://github.com/jannikbuschke/formik-mantine I use Formik a lot and even though Mantine provides a form state solution, I have several older apps that need to maintained/migrated and I am not yet sure if mantines solution is ready for me yet , or if I am ready to migrate. If anyone else is in the same shoes pls checkout the integration library above. Hope this is not off topic here

Can you add a demo for this? Thanks!

**Reply by jannikbuschke** [reply]
Added a sandbox and image
